purchase_create_related/models/purchase_order.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    can_be_secondary = fields.Boolean(default=False)
+
+    secondary_purchase_ids = fields.Many2many(
+        string="Secondary Purchases",
+        comodel_name="purchase.order",
+        relation="purchase_order_secondary_purchases",
+        column1="main_purchases",
+        column2="secondary_purchases",
+    )
+    secondary_purchases_count = fields.Integer(
+        compute="_compute_secondary_purchases_count"
+    )
+    main_purchase_ids = fields.Many2many(
+        string="Main Purchases",
+        comodel_name="purchase.order",
+        relation="purchase_order_secondary_purchases",
+        column1="secondary_purchases",
+        column2="main_purchases",
+    )
+    main_purchases_count = fields.Integer(compute="_compute_main_purchases_count")
+
+    @api.depends("main_purchase_ids")
+    def _compute_main_purchases_count(self):
+        for rec in self:
+            rec.main_purchases_count = len(rec.main_purchase_ids)
+
+    @api.depends("secondary_purchase_ids")
+    def _compute_secondary_purchases_count(self):
+        for rec in self:
+            rec.secondary_purchases_count = len(rec.secondary_purchase_ids)
+
+    def action_create_related_wizard(self):
+        not_allowed_purchases = self.filtered(lambda p: p.main_purchase_ids)
+        if not_allowed_purchases:
+            raise exceptions.ValidationError(
+                _(
+                    "The following purchases can not be linked to secondary purchases, "
+                    "due to already being a secondary purchase: {}"
+                ).format(",".join(not_allowed_purchases.mapped("name")))
```

Comment:
The comma could be separated.
Currently:
buy1,buy2,buy3,...
better
buy1, buy2, buy3, ...

Reply:
Done